Resizetizer.NT/Resizetizer.NT.targets

```diff
@@ -155,7 +155,7 @@
 		<!-- This allows us to invalidate the build based on not just input image files changing but project item metadata as well -->
 		<WriteLinesToFile
 			File="$(_ResizetizerInputsFile)"
-			Lines="@(SharedImage->'File=%(Identity);BaseSize=%(BaseSize);TintColor=%(TintColor)')"
+			Lines="@(SharedImage->'File=%(Identity);Link=%(Link);BaseSize=%(BaseSize);Resize=%(Resize);TintColor=%(TintColor);IsAppIcon=%(IsAppIcon);ForegroundScale=%(ForegroundScale);ForegroundFile=%(ForegroundFile)')"
```

Comment:
Added the rest of the properties that we care about. I kept getting skips when changing things.

Reply:
Yes!